### PR TITLE
feat(forms): add `Value.Selection` component

### PR DESCRIPTION
**Pull request by joakbjerk**
- [x] Fix and update tests
- [x] Improve and fix Value rendering

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection/info.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+showTabs: true
+---
+
+## Description
+
+`Value.Selection` is a wrapper component for displaying string values, with user experience tailored for selection values.
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection/info.mdx:7]:
```suggestion
`Value.Selection` is a component for displaying a string value based on a user selection.
```

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: 'Selection'
+description: '`Value.Selection` is a wrapper component for displaying string values, with user experience tailored for selection values.'
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection.mdx:3]:
```suggestion
description: '`Value.Selection` is a component for displaying a string value based on a user selection.'
```

**Review by tujoworker — APPROVED** [review]
Very nice work 🚀

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection/demos.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+<Examples.LabelAndValue />
+
+### Inline
+
+<Examples.Inline />
+
+{/* This is not yet supported */}
+{/* ### Field.Selection with path */}
+{/* When using the same `path` as on a `Field.Selection`, the title will be used as the displayed value. */}
+{/* <Examples.FieldSelectionPath /> */}
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection/demos.mdx:32]: Actually, this example should be possible to implement.

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: 'Selection'
+description: '`Value.Selection` is a component for displaying a string value based on a user selection.'
+componentType: 'base-value'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection.mdx:4]: What does this?
Never seen this prop before

File: packages/dnb-eufemia/src/extensions/forms/Value/Selection/SelectionDocs.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { PropertiesTableProps } from '../../../../shared/types'
+
+export const SelectionProperties: PropertiesTableProps = {
+  dataPath: {
+    doc: 'The path to the context data (Form.Handler). The object needs to have a `value` and a `title` property. The generated options will be placed above given JSX based children.',
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Value/Selection/SelectionDocs.ts:5]:
```suggestion
    doc: 'The path to the context data (Form.Handler). The context data object needs to have a `value` and a `title` property. The generated options will be placed above given JSX based children.',
```

When reading this in the [dos](https://eufemia-git-feat-value-selection-eufemia.vercel.app/uilib/extensions/forms/Value/Selection/properties/#field-specific-props), I don't 100% get it.

**Review by langz — APPROVED** [review]

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection.mdx (hunk 4 above)
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Selection.mdx:4]: Never mind :)

**Pull request merged**